custom/care_pathways/static/care_pathways/ko/report_filter.drilldown_options.js

```diff
@@ -148,9 +148,7 @@ var DrilldownOption = function (select, drilldown_map) {
 };
 
 $.fn.drilldownOptionFilter = function (options) {
-    this.each(function(i) {
-        var viewModel = new DrilldownOptionFilterControl(options);
-        ko.applyBindings(viewModel, $(this).get(i));
-        viewModel.init();
-    });
-};
\ No newline at end of file
+    var viewModel = new DrilldownOptionFilterControl(options);
+    $(this).koApplyBindings(viewModel);
```

Comment:
Does `$(this).length === 1`? Surprising given the previous use of `this.each()`. If it matches multiple, will `koApplyBindings` not throw a "Multiple elements passed to koApplyBindings" error?